src/common/io_exerciser/ObjectModel.cc

```diff
@@ -0,0 +1,162 @@
+#include "ObjectModel.h"
+
+ceph::io_exerciser::ObjectModel::ObjectModel(const std::string oid, uint64_t block_size, int seed) :
```

Comment:
This is a cpp file, so it makes sense to shorten the 'ceph::io_ex...'.
E.g.
using ObjectModel = ceph.....::objectmodel

Reply:
Sure, I can 'using' the namespaces in the cpp files in a few places throughout the pr if this is the preferred method, as they are fairly long in places.